setup.py

```diff
@@ -13,7 +13,7 @@
     description='Airbrake exception logger for Django',
     license='MIT',
     long_description=open('README.rst').read(),
-    install_requires=['Django>=1.8,<1.9.99'],
+    install_requires=['Django>=1.8,<=1.11.11'],
```

Comment:
I used 1.9.99 to specify all version up to/excluding 1.10 (and pre-releases of 1.10). Also specifying 1.11.11 is problematic when a bugfix is released (1.11.12) which would be excluded.

As 1.8 has almost reached EOL (April 1st), we can specify 1.11 as the lower bound and not have an upper bound.